### ISSUE TITLE
Graph rendering differs on branching

### ISSUE THREAD

**Issue by nicoespeon** [nicoespeon/gitgraph.js#174]
From #159.

Graph does not render in the same way if a commit is made before or after branching?

![image](https://user-images.githubusercontent.com/1094774/29156538-c4dbb456-7da2-11e7-8e32-dc0d3fa19f40.png)

https://jsfiddle.net/Laymain/qd2q91pe/

**Comment by yodalee** [comment]
I found that, if remove the "mode: compact", the render will be OK. This may give us a hint where the problem is.

**Comment by yodalee** [comment]
https://github.com/nicoespeon/gitgraph.js/blob/develop/src/gitgraph.js#L685-L692
I think this may be a feature rather than bug w

**Comment by nicoespeon** [comment]
From my tests, this will be fixed with #187.

This can be tested with following code:

```js
const master = gitgraph.branch("master");
master.commit();
const develop = gitgraph.branch("develop");

// Swapping these 2 lines should not change rendered graph
master.commit();
develop.commit();

master.merge(develop);
```

**Comment by nicoespeon** [comment]
So, this is technically _not_ resolved for the `gitgraph.js` package… But we released `@gitgraph/react` in v1.0.0-0 and this issue was fixed here.

Thus, I'm closing this issue as we won't maintain the `gitgraph.js` package. There will be a `@gitgraph/js` package soon to replace it. 

We still accept PRs if someone really needs it, but I'd suggest moving to the aforementioned packages. Thanks 🙏
